sourcecode/hub/resources/views/content/show.blade.php

```diff
@@ -1,27 +1,142 @@
 <x-sidebar-layout>
     <x-slot:title>{{ $content->latestPublishedVersion->resource->title }}</x-slot:title>
 
+    <div class="container mt-1">
+        <div class="row">
+            <div class="col-6">
+                {{ trans('messages.created')}}: {{ $content->created_at->format('M d, Y') }} by {{ $authorName }}
```

Comment:
The data format is different depending on the language/locale of the user. Laravel casts dates to Carbon instances, so you can use `isoFormat` and format should be `L` or `LL`.
Se the [Carbon documentation](https://carbon.nesbot.com/docs/#api-localization) on localization for more info